runtime/doc/lsp.txt

```diff
@@ -763,19 +763,27 @@ client()                                                      *vim.lsp.client*
         before text is sent to the server.
       • {handlers} (table): The handlers used by the client as described in
         |lsp-handler|.
+      • {commands} (table): Table of command name to function which is called
+        if any LSP action (code action, code lenses, ...) triggers the
+        command. Client commands take precedence over the global command
+        registry.
       • {requests} (table): The current pending requests in flight to the
         server. Entries are key-value pairs with the key being the request ID
         while the value is a table with `type`, `bufnr`, and `method`
         key-value pairs. `type` is either "pending" for an active request, or
         "cancel" for a cancel request. It will be "complete" ephemerally while
         executing |LspRequest| autocmds when replies are received from the
         server.
-      • {config} (table): copy of the table that was passed by the user to
-        |vim.lsp.start_client()|.
+      • {config} (table): reference of the table that was passed by the user
```

Comment:
Could we have a more specific type for `config`?

Reply:
We don't have any specific types in this block and for now I don't really want to mess with it.

Soon on my todo will be to generate vimdoc from `@class` blocks, at which point it will make sense to replace all these types.